stock_picking_invoicing/stock.py

```diff
@@ -49,3 +49,19 @@ def set_to_be_invoiced(self):
                                   (picking.name, picking.invoice_id.number))
                 picking.invoice_state = '2binvoiced'
         return True
+
+
+class StockMove(models.Model):
+    _inherit = 'stock.move'
+
+    @api.model
+    def _get_master_data(self, move, company):
+        data = super(StockMove, self)._get_master_data(move, company)
+        if move.picking_id.partner_id.id != data[0].id:
+            # if someone else modified invoice partner, I use it
+            return data
+        partner_invoie_id = move.picking_id.partner_id.address_get(
```

Comment:
On 7 July 2015 at 17:28, Alex Comba <notifications@github.com> wrote:

> It would be better partner_invoice_id. Could you please rename it?
>
>
>
Fixed, thanks